vesta.php

```diff
@@ -1463,6 +1463,37 @@ private function generateUsername($domain)
         }
 
         $username = substr($username, 0, min($length, 8));
+
+        // Check for an existing user account
+        $row = $this->getModuleRow();
+
+        $vesta = null;
+        if ($row) {
```

Comment:
Can we handle the case where there is no module row more eloquently? Perhaps an else statement to return the `$username` we originally constructed? It would be great to avoid potential non-object errors from method calls like line 1482.